apps/condo/domains/common/components/Comments/CommentsTabContent.tsx

```
@@ -0,0 +1,186 @@
+                            },
+                        },
+                    })
+                }
+
+                return (
+                    <>
+                        <Comment
+                            comment={comment}
+                            deleteAction={deleteAction}
+                            setEditableComment={setEditableComment}
+                        />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)